requirements-dev.txt

```diff
@@ -31,6 +31,7 @@ pycodestyle==2.5.0
 pydruid==0.6.1
 pyhive==0.6.2
 pylint==2.5.3
+ded thrift dependency
```

Comment:
typo?

Reply:
Whoah! That's a good catch, yes that's a typo. I'll fix it